kge/config-default.yaml

```diff
@@ -327,7 +331,8 @@ eval:
   # mean_reciprocal_rank_filtered_with_test.
   filter_with_test: True
 
-  # Type of evaluation (entity_ranking only at the moment)
+  # Type of evaluation (entity_ranking or training_loss). Currently, 
+  # entity_ranking runs training_loss as well.
```

Comment:
I vote for disabling the automatic running of training_loss. Makes evaluation too costly.

Reply:
Validation and test data are a lot smaller (1/10 is the worst valid/train ratio in our current datasets).